editoast/src/railjson/operation/delete.rs

```diff
@@ -81,4 +79,70 @@ mod test {
             Ok(())
         });
     }
+
+    #[test]
+    fn delete_signal() {
+        let conn = PgConnection::establish(&PostgresConfig::default().url()).unwrap();
+        conn.test_transaction::<_, Error, _>(|| {
+            let signal_creation = RailjsonObject::Signal {
+                railjson: Signal {
+                    ..Default::default()
+                },
```

Comment:
```suggestion
                railjson: Default::default(),                
```

If you're not changing any fields from the default value then you can simply use it.